lib/clearance/configuration.rb

```diff
@@ -93,6 +98,7 @@ def initialize
       @cookie_name = "remember_token"
       @httponly = false
       @mailer_sender = 'reply@example.com'
+      @password_reset_time_limit = 15.minutes
```

Comment:
@mxie Did you base the default on anything other libraries are doing for this? I'm wondering if it might be a bit too low, but I don't really know. I know that when I have to reset my password, sometimes the email doesn't come immediately and I get distracted doing something else. Maybe `1.hour`? I don't feel strongly and can easily be convinced 15 minutes is correct.